### DISCUSSION TITLE
offset and duration options?

### DISCUSSION THREAD

**Discussion by RealSting** [Purfview/whisper-standalone-win#343] [Q&A] [answered]
Hi!
are there any options like on CPP version? 
```
-ot N,     --offset-t N        [0] time offset in milliseconds
-d  N,     --duration N        [0] duration of audio to process in milliseconds
```
Thanks!

**Comment by Purfview** [comment]
I don't know what "offset ad duration" means nor what those cpp options do.

**Reply by RealSting** [reply]
They are options.
If you want Whisper writes only from 1h:05m for 5s you can specify `-ot 3900000 -d 5000` without transcribe other parts of a movie

**Reply by Purfview** [reply, marked as answer]
Use `--clip_timestamps` / `-clip` - Comma-separated list start,end,start,end,... timestamps (in seconds) of clips to process:

Example with round seconds -`-clip 3900,3905`, with fraction`-clip 3900.4,3905.9`, with few clips - `-clip 3900,3905,4567,4572`
